cmd/info_cmd.go

```diff
@@ -247,3 +258,20 @@ func showTx(ctx *cli.Context) error {
 	PrintJsonObject(txInfo)
 	return nil
 }
+
+func getShardChainConfig(ctx *cli.Context) error {
+	SetRpcPort(ctx)
+	if ctx.NArg() < 2 {
+		PrintErrorMsg("Missing argument. shardID,height expected.")
+		cli.ShowSubcommandHelp(ctx)
+		return nil
+	}
+	shardID, _ := strconv.ParseUint(ctx.Args().First(), 10, 32)
+	height, _ := strconv.ParseUint(ctx.Args().Get(1), 10, 32)
+	chainConfig, err := utils.GetShardChainConfig(uint64(shardID), height)
```

Comment:
shardId does not need to cast to uint64, and height param should be uint32

Reply:
yes